app/javascript/mastodon/features/notifications_v2/components/embedded_status.tsx

```diff
@@ -96,12 +98,17 @@ export const EmbeddedStatus: React.FC<{ statusId: string }> = ({
     [],
   );
 
+  const handleContentWarningClick = useCallback(() => {
+    setExpanded((v) => !v);
+  }, [setExpanded]);
+
```

Comment:
Shouldn't it rather use the shared redux state?

Reply:
I'm not actually sure that would be welcome or useful.